Solutions/1Password/README.md

```diff
@@ -0,0 +1,187 @@
+# 1Password (Preview)
+
+## Overview
+
+The key function of this Solution is to retrieve sign-in attempts, item usage, and audit events logs from your 1Password Business account using the 1Password Events Reporting API, and store it in an Azure Log Analytics Workspace using Microsoft cloud native features.
+
+## Azure services needed
+
+### Required
+
+- [1Password Business account](https://1password.com/business)
+- [1Password Events API key](https://support.1password.com/events-reporting/#appendix-issue-or-revoke-bearer-tokens)
+- [Microsoft Azure](https://azure.microsoft.com/en-us/free)
+- [Microsoft Sentinel](https://azure.microsoft.com/en-us/products/microsoft-sentinel/)
+- Contributor role with User Access Administrator role on the Microsoft Sentinel Resource Group <br>
+**or**
+- Owner on the Microsoft Sentinel Resource Group 
+
+## Automated Installation
+
+Installing the 1Password Solution for Microsoft Sentinel is easy and can be completed in only a few minutes. 
+Just click the button below to get started with the deployment wizard. <br>
+
+[![Deploy To Azure](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/deploytoazure.svg?sanitize=true)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fazurekid%2FAzure-Sentinel%2Ffeature%2F1password%2FSolutions%2F1Password%2FData%20Connectors%2F1Password%2Fazuredeploy_1Password_API_FunctionApp.json/createUIDefinitionUri/https%3A%2F%2Fraw.githubusercontent.com%2Fazurekid%2FAzure-Sentinel%2Ffeature%2F1password%2FSolutions%2F1Password%2FData%20Connectors%2F1Password%2Fdeployment%2FUiDefinition.json)
```

Comment:
The link used in this is from your repo. It should consist Master branch's address

Reply:
fixed, but still an issue in the pipeline regarding links. please advice

Reply:
replaced the link to reference azure/azure-sentinel/master/...